src/main/java/net/e621/mcumbrella/antishulkerdispensercrash/A.java

```diff
@@ -35,8 +59,8 @@ public void onSDC(TickBlockEvent e)
                 (e.getTargetBlock().getPosition().getY()==0&&e.getTargetBlock().getExtendedState().get(Keys.DIRECTION).get()==Direction.DOWN))
         )
         {
-            e.setCancelled(true);
-            l.warn("Blocked possible shulker box dispenser crash at: X="+e.getTargetBlock().getPosition().getX()+", Y="+e.getTargetBlock().getPosition().getY()+", Z="+e.getTargetBlock().getPosition().getZ());
+            Inventory i=(Inventory)e.getTargetBlock().getLocation().get().getTileEntity().get();
+            for(ItemType b:blkList){if(i.contains(b)){e.setCancelled(true);l.warn("Blocked possible shulker box dispenser crash at: X="+e.getTargetBlock().getPosition().getX()+", Y="+e.getTargetBlock().getPosition().getY()+", Z="+e.getTargetBlock().getPosition().getZ());break;}}
```

Comment:
reduce readability while reducing size - my consistent style when officially releasing the code.
I can also merge line62 into line63 but I'm worried that it may call many same things multiple times and then, slow down the running speed. the new detection has almost doubled the processing time of this event bruh